api/authentication/patron_authentication_provider.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from werkzeug.datastructures import Authorization
+
+from api.authentication.access_token import AccessTokenProvider
+from api.authentication.base import AuthenticationProvider
+from api.problem_details import PATRON_AUTH_ACCESS_TOKEN_INVALID
+from core.model import Patron, Session, get_one
+from core.util.problem_detail import ProblemDetail
+
+
+class PatronAccessTokenAuthenticationProvider(AuthenticationProvider):
+    FLOW_TYPE = "http://librarysimplified.org/authtype/bearer"
```

Comment:
This is an imaginary flow type, if there are semantics to this we should change it.

Reply:
I think the mobile clients will inspect this value to decide whether a server supports token auth, right?

If so, yeah, we should decide if this is the constant we want. Extremely hard to change afterwards.